src/pages/ReimbursementAccount/BankAccountPlaidStep.js

```diff
@@ -103,6 +116,27 @@ class BankAccountPlaidStep extends React.Component {
                         bankAccountID={bankAccountID}
                         selectedPlaidAccountID={selectedPlaidAccountID}
                     />
+                    {Boolean(selectedPlaidAccountID) && !_.isEmpty(lodashGet(this.props.plaidData, 'bankAccounts')) &&
+                        <CheckboxWithLabel
+                            style={styles.mt4}
+                            inputID="acceptTerms"
+                            LabelComponent={() => (
+                                <Text>
+                                    {this.props.translate('common.iAcceptThe')}
+                                    <TextLink
+                                        href="https://use.expensify.com/terms"
+
+                                        // to call the onPress in the TextLink before the input blur is fired and shift the link element
+                                        onMouseDown={e => e.preventDefault()}
```

Comment:
Is this needed here?
This fix was added to resolve https://github.com/Expensify/App/issues/12715, could you double-check if the bug is present on this page without this line?

Reply:
Please add steps from https://github.com/Expensify/App/pull/12940 to make sure we don't break it